enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -633,18 +724,30 @@ customizing these parameters.
 For the time being, we may defer the ability to customize the parameters
 described above and address the question in a followup EP for tech preview.
 
-**Resolution**: TBD.
+**Resolution**: Deferred until tech preview.  For dev preview, we do not allow
+customization of the SMCP.  The Ingress Operator simply specifies the
+"gateway-controller" profile.  This profile enables Gateway API for ingress,
+with cluster-wide watches for Gateway API CRs.  It also enables mesh, and
+individual namespaces can be added to this mesh using ServiceMeshMember and
+ServiceMeshMemberRoll CRs.  We will revisit the possibility of allowing the
+cluster admin to configure parameters on the SMCP in tech preview once we have
+some end-user feedback regarding what parameters end-users want to be able to
+customize on the SMCP.
```

Comment:
Did I miss where you mention whether a namespace can both be used by gateways and be a part of the service mesh?  Or is that a different question?

Reply:
I don't think that that question has come up before.  Is there a reason to believe it might not be possible for a namespace both to be in a mesh and to have routes attached to a gateway?

Reply:
It's worth testing whether there would be two envoy pods installed, and whether they conflict somehow.  But the details don't need to be worked out right now.

Reply:
It depends on how the SMCP is configured.  The Ingress Operator configures the SMCP with `spec.gateways.ingress.enabled: true`, which results in having the "istio-ingressgateway" deployment, and the Ingress Operator also enables Gateway API, which has automated deployment, so there should be an Envoy deployment for each gateway.  Then (once https://issues.redhat.com/browse/OSSM-3358 is fixed) the cluster admin can create an SMCP that creates Envoy deployments as specified using `spec.gateways.ingress.enabled` and `spec.gateways.egress.enabled`.  If there were some conflict between Envoy deployments, that would be a bug in OSSM.